src/App.js

```diff
@@ -1,18 +1,21 @@
 import { Routes, Route } from "react-router-dom";
 
+import { useVideos } from 'context'
 import {
   HomePage,
   PlaylistPage,
   WatchLaterPage,
   LikedPage,
   HistoryPage,
   Login,
-  SignUp
+  SignUp,
+  SingleVideoPage
 } from 'pages'
 
 import "./App.css";
 
 function App() {
+  const { videos } = useVideos()  

```

Comment:
I think you are not using it anywhere in the code. So maybe you can remove it.

Reply:
Thanks, removed it.